File: 1014 - consumo/consumo.scala
object Main{
    def consumo (x : Int, y : Double) : Double =
        x / y

    def main( args: Array[String]){
        var x = scala.io.StdIn.readInt() : Int
        var y = scala.io.StdIn.readDouble() : Double
        var res = consumo(x , y)
        printf("%.3f km/l\n", res)
    }
}

File: 1014 - consumo/consumo.pas
program consumo;

uses Sysutils;

function consumo( x : integer; y : real) : real;
begin
    consumo := x / y;
end;

var 
    x : integer;
    y, res : real;

begin
    readLn(x);
    readLn(y);
    res := consumo(x , y);
    WriteLn (Format('%.3f',[res]), ' km/l');
end.